src/server/transaction.cc

```diff
@@ -1295,18 +1300,18 @@ void Transaction::MultiReportJournalOnShard(EngineShard* shard) const {
   }
 }
 
-void Transaction::UnlockMultiShardCb(absl::Span<const std::string_view> sharded_keys,
-                                     EngineShard* shard) {
+void Transaction::UnlockMultiShardCb(absl::Span<const LockFp> fps, EngineShard* shard) {
   DCHECK(multi_ && multi_->lock_mode);
 
   MultiReportJournalOnShard(shard);
 
   if (multi_->mode == GLOBAL) {
     shard->shard_lock()->Release(IntentLock::EXCLUSIVE);
   } else {
-    for (const auto& key : sharded_keys) {
-      shard->db_slice().ReleaseNormalized(*multi_->lock_mode, db_index_, key);
-    }
+    KeyLockArgs lock_args;
+    lock_args.db_index = db_index_;
+    lock_args.fps = fps;
```

Comment:
What's wrong with `KeyLockArgs lock_args{db_index_, fps}` ?

Reply:
I do not think you can do it because of the default initializer with the third variable.

Reply:
now we can do it